CHANGELOG.md

```diff
@@ -38,6 +38,23 @@
   [Otávio Lima](https://github.com/otaviolima)
   [#1710](https://github.com/realm/SwiftLint/issues/1710)
 
+## 0.22.0: Coffee Maker
+
+##### Breaking
+
+* None.
+
+##### Enhancements
+
+* None.
+
+##### Bug Fixes
+
+* Fix the warning message of `vertical-whitespace` rule to display the maximum empty lines allowed if `maxEmptyLines` is greater than 1.
+* Fix the autocorrection of `vertical-whitespace` rule to keep the `maxEmptyLines` in the cleaned section.
```

Comment:
requires two trailing spaces as described in CONTRIBUTING.md: https://github.com/realm/SwiftLint/blob/master/CONTRIBUTING.md#tracking-changes